source/Calamari.Common/Plumbing/Extensions/LogExtensions.cs

```diff
@@ -23,5 +23,16 @@ public static void LogMetric(this ILog log, string metricName, object metricValu
 
             log.WriteServiceMessage(new ServiceMessage(ServiceMessageNames.CalamariDeploymentMetric.Name, serviceMessageParameters));
         }
+
+        public static void AddValueToRedact(this ILog log, string? value, string replacement)
```

Comment:
See my comment on `RedactedValuesLogger.cs` regarding the nullable value.

Reply:
Fixed